src/Easify.Azure.AspNetCore.UnitTests/Easify.Azure.AspNetCore.UnitTests.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
why didnt we do this in net 5?

Reply:
Discussed with Mo, leave this as per 3.1.